man1/iconv.rst

```diff
@@ -0,0 +1,107 @@
+İSİM
+====
+
+iconv - metni bir karakter kodlamasından diğerine dönüştürür
+
+ÖZET
+========
+
+**iconv** [* seçenekler *] [-f *from-encoding*] [-t *to-encoding*]
+[*inputfile*]...
+
+AÇIKLAMA
+===========
+
+**iconv** programı bir kodlamadaki metni okur ve başka bir kodlamadaki metni verir. Hiçbir giriş dosyası verilmezse veya kısa çizgi (-) olarak verilirse, **iconv** standart girişten okunur. Hiçbir çıktı dosyası verilmezse, **iconv** standart çıktıya yazar.
+
+* From-encoding * belirtilmezse, varsayılan değer geçerli yerel ayarın karakter kodlamasından türetilir. * To-encoding * belirtilmezse, varsayılan değer geçerli yerel ayarın karakter kodlamasından türetilir.
```

Comment:
* From-encoding * kısmında rst formatından çıkış var  Ayrıca * To-encoding * için de.

"*ifade*" formatı yazıyı italik yapmak içindir lütfen dikkat edelim.